qwandaq/src/main/java/life/genny/qwandaq/utils/QwandaUtils.java

```diff
@@ -1114,22 +1070,22 @@ public Attribute getAttributeByBaseEntityAndCode(BaseEntity baseEntity, String a
 	 * @param value     The value to check
 	 * @return Boolean representing whether the validation conditions have been met
 	 */
-	public Boolean validationsAreMet(Attribute attribute, String value) {
+	public static Boolean validationsAreMet(Attribute attribute, String value) {
```

Comment:
Prepping this for unit testing, this also does not need to be an instance method